pkg/services/thumbs/dashboardthumbsimpl/dashboard_thumbs.go

```diff
@@ -0,0 +1,43 @@
+package dashboardthumbsimpl
```

Comment:
What's the purpose of this file? Can't `xormStore` register itself as the `thumbs.DBThumbService`? I assume alternative implementation (such as those sqlx-based) would implement all methods present in the `thumbs.DBThumbService` interface, so I am not sure if we need this extra layer of abstraction